### PR TITLE
Add Contributing guidelines and add a link to them from the README

### PR DESCRIPTION
**Pull request by SueChaplain**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+## Contributing to www.kitura.io
+
+Content on this site is licensed under the Apache Licence, Version 2.0.
+
+**Important**: If you clone this site to a Windows system you must configure git to handle the correct line ending characters by running the following command:
+'git config --global core.autocrlf true`
```
Comment (ianpartridge) [inline CONTRIBUTING.md]: Should this line start with a backtick, not a single quote?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+If you want to add topics or update existing topics, please open an Issue or create a Pull request with your changes.
+Make sure you use the page language variable on all your links by appending `/{{ page.lang }}`. For example, to link to the *Getting Started* topic, use:
+	`/{{ page.lang }}/starter/gettingstarted.html`
+
+You might also like to make contributions in other languages, by translating the English content.
+
+Follow these steps:
+
+1. Clone the kitura.io repository
+2. Find the 2-digit ISO 639-1 code for the language you wish to translate into.
```
Comment (ianpartridge) [inline CONTRIBUTING.md]: Where would people go to find this? Can we link them somewhere?

File: CONTRIBUTING.md (hunk 1 above)
Comment (SueChaplain) [inline CONTRIBUTING.md]: Yes - didn't have my glasses on!

Comment (ianpartridge) [inline CONTRIBUTING.md]: Thanks. Did that stop it rendering monospaced?

**Comment by ianpartridge** [comment]
I think we need to add `CONTRIBUTING.md` to the exclude list at https://github.com/IBM-Swift/kitura.io/blob/master/_config.yml#L9

Otherwise it will be rendered by Jekyll at `kitura.io/contributing.html`

**Pull request merged**